### PR TITLE
MMT MPU Generation

### PR DESCRIPTION
**Pull request by DerouineauNicolas**
Tested with 

MP4Box -add ~/Vidéos/small.mp4 -mmt-seq-num 10 -mmt-asset-id-scheme URI -mmt-asset-id-value http://www.gpac.io mpu10.mp4

MP4Box -add ~/Vidéos/small.mp4 -mmt-seq-num 12 -mmt-asset-id-scheme URI -mmt-asset-id-value http://www.gpac.io mpu10.mp4

If the input contains N tracks, 

MP4Box -add ~/Vidéos/VideoandAudio.mp4 -mmt-autogen -frag 100 mpu.mp4

N Files will be generated. For this case, UUID are Hardcoded values . 

Once generated, mpu boxes can be parsed with 

MP4Box -std -diso mpu11.mp4

### CODE REVIEW

File: applications/mp4box/main.c [hunk 1]
```diff
@@ -1850,6 +1852,12 @@ const char *grab_m2ts = NULL;
 const char *grab_ifce = NULL;
 #endif
 FILE *logfile = NULL;
+Bool mpu_is_complete=0;
```
Comment (cconcolato) [inline applications/mp4box/main.c]: Why use global variable ?

Comment (DerouineauNicolas) [inline applications/mp4box/main.c]: Is there another way to retrieve the parameters found in mp4box_parse_args function ?

Comment (jeanlf) [inline applications/mp4box/main.c]: nope that's the right way...

**Comment by rbouqueau** [comment]
1) looks like this variable is unused, is this expected?
2) can't we put this global vars as static to limit the scope?

Romain Bouqueau
CEO @ www.gpac-licensing.com
romain.bouqueau@gpac-licensing.com
+33 (0) 614 685 991
skype : romain.bouqueau

2016-09-21 23:34 GMT+02:00 jeanlf <notifications@github.com>:

**Comment by DerouineauNicolas** [comment]
@rbouqueau: 
1, Yes, this parameter is not used at the moment, as this is only preliminary investigation.
2, You're right, I'll set them as static

**Pull request closed without merging**